installation/resources/readme.md

```diff
@@ -0,0 +1,22 @@
+# Compass Resources
+
+## Installation CRs
+
+### [Kyma Dependencies](installer-cr-kyma-dependencies.yaml)
```

Comment:
I think a list or a table would look better than having separate heading for each item.